blazor/datagrid/cell-edit-types.md

```diff
@@ -984,6 +984,71 @@ The `SfTextBox` component is defined inside the EditTemplate and changes can be
 }
 ```
 
+### Prevent enter key functionality in multiline textbox while editing
+
+By default, the ENTER key functionality in the Grid will save the edited changes. In Multiline Textbox, the ENTER key should add a new line break instead of saving the edited changes while editing. This can be achieved by using Microsoft JsInterop.
```

Comment:
pressing the ENTER key will save the changes in Grid when focus is in Grid Edit form (both normal or dialog)